security/firely-auth/firely-auth-settings.rst

```diff
@@ -358,6 +359,7 @@ You register a :term:`client` in the ``AllowedClients`` array. For each client y
 - ``AllowFirelySpecialScopes``: true / false - Allow app to request scopes for Firely Server specific operations. Currently just 'http://server.fire.ly/auth/scope/erase-operation'
 - ``RequireClientSecret``: true / false - A :term:`public client` cannot hold a secret, and then this can be set to ``false``. Then the ``ClientSecrets`` section is ignored. See also the note below.
 - ``RefreshTokenLifetime``: If the client is allowed to use a :term:`refresh token`, how long should it be valid? The value is in days. You can also use HH:mm:ss for lower values.
+- ``ConsentLifetime`` : This is an optional setting which can specify a period after which the users consent will be valid. The value is in days. You can also use HH:mm:ss for lower values.
```

Comment:
Should it not be:
This is an optional setting which can specify a period after which the users consent will be **revoked.**?